postgres-12/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/postgres-12/postgres-12.tmpl.dockerfile.
-FROM postgres:12.7
-ARG VERSION=10.0.4
+FROM postgres:12.6
```

Comment:
Why the downgrade? Is this because of the template used in the scripts?

Reply:
Fixed